app/javascript/components/public_health/AdvancedFilter.js

```diff
@@ -0,0 +1,608 @@
+          type: 'option',
+          options: ['Unknown', 'E-mailed Web Link', 'SMS Texted Weblink', 'Telephone call', 'SMS Text-message', 'Opt-out'],
+        },
+        { name: 'latest-report', title: 'Latest Report', description: 'Monitorees with latest report', type: 'date' },
+        { name: 'hoh', title: 'Head of Household', description: 'Monitorees that are a head of household', type: 'boolean' },
+        { name: 'enrolled', title: 'Enrolled', description: 'Monitorees enrollment', type: 'date' },
+        {
+          name: 'last-date-exposure',
+          title: 'Last date of exposure',
+          description: 'Monitorees who have a last date of exposure',
+          type: 'date',
+        },
+        { name: 'symptom-onset', title: 'Symptom onset', description: 'Monitorees who have a symptom onset', type: 'date' },
+        { name: 'continous-exposure', title: 'Continuous Exposure', description: 'Monitorees who have continuous exposure enabled', type: 'boolean' },
+        { name: 'telephone-number', title: 'Telephone Number', description: 'Monitoree telephone number', type: 'search' },
+        { name: 'email', title: 'Email', description: 'Monitoree email address', type: 'search' },
+        { name: 'sara-id', title: 'Sara Alert ID', description: 'Monitoree Sara Alert ID', type: 'search' },
+        { name: 'first-name', title: 'First Name', description: 'Monitoree first name', type: 'search' },
+        { name: 'middle-name', title: 'Middle Name', description: 'Monitoree middle name', type: 'search' },
+        { name: 'last-name', title: 'Last Name', description: 'Monitoree last name', type: 'search' },
+        {
+          name: 'monitoring-plan',
+          title: 'Monitoring Plan',
+          description: 'Monitoree monitoring plan',
+          type: 'option',
+          options: [
+            'None',
+            'Daily active monitoring',
+            'Self-monitoring with public health supervision',
+            'Self-monitoring with delegated supervision',
+            'Self-observation',
+          ],
+        },
+        { name: 'never-responded', title: 'Never Responded', description: 'Monitorees who have never reported', type: 'boolean' },
+        {
+          name: 'risk-exposure',
+          title: 'Risk Exposure',
+          description: 'Monitoree risk exposure',
+          type: 'option',
+          options: ['High', 'Medium', 'Low', 'No Identified Risk'],
+        },
+        { name: 'require-interpretation', title: 'Requires Interpretation', description: 'Monitorees who require interpretation', type: 'boolean' },
+        {
+          name: 'preferred-contact-time',
+          title: 'Preferred Contact Time',
+          description: 'Monitoree preferred contact time',
+          type: 'option',
+          options: ['Morning', 'Afternoon', 'Evening'],
+        },
+      ],
+      savedFilters: [],
+      activeFilter: null,
+      applied: false,
+    };
+    this.add = this.add.bind(this);
```

Comment:
I think that change should be made globally at once. Since the vast majority of the code base still does it this way, I'd rather stick with this until that time. I don't really think space saving is a good reason to switch though, so let's think about it.